### ISSUE TITLE
Update to Guzzle 4.x for HHVM support

### ISSUE THREAD

**Issue by seeekr** [keenlabs/KeenClient-PHP#64]
Is anyone else using HHVM and wishing that the Keen IO PHP client would work on top of that? I'm not sure what else needs to be done, but Guzzle could be updated to 4.x (or even 5.x) and that would be one step towards supporting HHVM.
Are there any plans or work already underway?

**Comment by nicklee1990** [comment]
just checking in on this issue, are there any plans to update and move to a more recent version of guzzle i.e. 5.* or 6.*?

**Comment by semiosis** [comment]
Any thoughts on updating to guzzle 6?

**Comment by caseysoftware** [comment]
FYI: I looked into upgrading to Guzzle 6 recently.

The Service Description class has been removed from core and split into [Guzzle Service](https://github.com/guzzle/guzzle-services) and the [Guzzle Description Loader](https://github.com/gimler/guzzle-description-loader), neither of which have been updated to Guzzle 6.

I just submitted #85 which bumps the dependencies (technically, unnecessary) so they're compatible with G6. Once that's ready, it should be a (hopefully) painless transplant.
